values-hub.yaml

```diff
@@ -23,6 +23,12 @@ clusterGroup:
       namespace: nvidia-gpu-operator
       channel: v23.6
       source: certified-operators
+    edb:
+      name: cloud-native-postgresql
+      namespace: openshift-operators
+      channel: stable-v1.23
+      source: certified-operators
+
```

Comment:
The reason this was not added in values-hub.yaml is, we are using the same validated pattern to deploy REDIS and in that case we don't want to deploy EDB operator. Can you update the PR to remove the deploying of EDB by default

Reply:
The reason for this is that the order matters, previously, if the operator isn't installed first nothing is deployed since the argocd workflow just get stuck and nothing is deployed, so, can we set some kind of order so in that chart the operator is deployed before anyone ?